roles/ocp_on_libvirt/templates/hosts.j2

```diff
@@ -86,8 +90,5 @@ ansible_ssh_extra_args='-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/n
 #registry_source_mirrors=["quay.io/openshift-release-dev/ocp-v4.0-art-dev", "registry.svc.ci.openshift.org/ocp/release", "quay.io/openshift-release-dev/ocp-release"]
 
 {% if enable_redfish | bool -%}
```

Comment:
Not sure I understand the purpose of this conditional if nothing is done, should we remove it instead?

Reply:
Sorry, I removed some code from there by mistake. Put it back.